snappy_wrappers/wrappers/bbduk/run/wrapper.py

```diff
@@ -177,6 +177,9 @@
         skipr2={config[skipr2]}                       \
         ecco={config[ecco]}                           \
         ktrim={config[ktrim]}                         \
+        $(if [[ -n "{config[kmask]}" ]] ; then \
+            echo kmask={config[kmask]} 
+        fi) \
         kmask={config[kmask]}                         \
```

Comment:
Shouldn't you remove the line without the if check here ?

Reply:
You're absolutely right, thanks!